tools/harness-thci/ARM.py

```diff
@@ -2190,6 +2203,7 @@ def MGMT_COMM_SET(self, Addr='ff02::1', xCommissionerSessionID=None, xSteeringDa
             ModuleHelper.WriteIntoDebugLogger("MGMT_COMM_SET() Error: " + str(e))
 
     def setActiveDataset(self, listActiveDataset=[]):
+        print '%s call setActiveDataset' % self.port
         pass
```

Comment:
just for debug purpose, sometimes it help to know which test case call this API.